gadgets/traceloop/README.mdx

```diff
@@ -8,38 +8,79 @@ import TabItem from '@theme/TabItem';
 
 # traceloop
 
-The traceloop gadget is a syscalls flight recorder.
+The trace_loop gadget is used to capture and replay system calls in real-time, acting as a flight recorder for your Kubernetes applications.
 
 ## Requirements
 
-- Minimum Kernel Version : 5.10
+- Minimum Kernel Version: 5.10+
+- Dependencies: Inspektor Gadget installed
+- Access: kubectl access to your cluster
```

Comment:
Other gadgets don't include this. I am not sure it is worth mentioning. And also, kubectl is only in the Kubernetes case.